src/domain/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx

```diff
@@ -131,3 +134,46 @@ test.each([1, 4])(
     });
   }
 );
+
+it.each([
```

Comment:
Types could be defined here to avoid type assertions in tests:

`it.each<[number, Language, EventType, string]>([`